tests/staking/observable/createWithdrawTx.test.ts

```diff
@@ -0,0 +1,169 @@
+import { ObservableStaking } from "../../../src";
+import { testingNetworks } from "../../helper";
+import * as stakingScript from "../../../src/staking/stakingScript";
+import * as observableStakingUtils from "../../../src/staking/observable";
+import * as staking from "../../../src/staking";
+
+describe("Create withdraw early unbonded transaction", () => {
```

Comment:
```suggestion
describe("Create withdrawal transactions", () => {
```

as we're testing both early unbonded and natural expiration here